Solutions/Vellum.Cli.Abstractions/Vellum.Cli.Abstractions.csproj

```diff
@@ -32,12 +31,4 @@
     <NoWarn>RCS1029; CS1591; SA1600;</NoWarn>
   </PropertyGroup>
 
-  <ItemGroup>
```

Comment:
How come the analyzer packages are removed?

Reply:
I think this is a glitch that they should be coming through via a RecommendedPractices packages, but there was a bug where those transitive dependencies would get upgraded outside of that and then would get added into the csproj